migrations/env.py

```diff
@@ -0,0 +1,53 @@
+from alembic import context
+from flask import current_app
+from sqlalchemy import engine_from_config, pool
+from logging.config import fileConfig
+import logging
+
+config = context.config
+
+fileConfig(config.config_file_name)
+logger = logging.getLogger('alembic.env')
```

Comment:
this is not a logger coming from the `alembic.env` module but used in our own code instead, I would go for `getLogger(__name__)`

Reply:
This directory is generated code.

Reply:
Wonder if we can actually exclude it...

Reply:
Doesn't look likely. 😞

Reply:
no, you should be able to do a `logging.getLogger('alembic.env').setLevel(logging.ERROR)` (or something similar - we do it to quieten boto)

Reply:
ok for silencing it if needed, on the other hand if it's code generated by the `alembic.env` module it makes sense it logs as such